src/Layout/redist/targets/GeneratePackagePruneData.targets

```diff
@@ -0,0 +1,60 @@
+<Project>
+
+  <Target Name="GeneratePackagePruneData">
+
+    <GeneratePackagePruneDataDownloads
+          NETCoreAppTargetFrameworkVersion="$(MicrosoftNETCoreAppRuntimePackageVersion)">
+       <Output TaskParameter="TargetingPackDownloads" ItemName="TargetingPackForPruneData" />
+    </GeneratePackagePruneDataDownloads>
+
+    <CollatePackageDownloads Packages="@(TargetingPackForPruneData)">
+      <Output TaskParameter="PackageDownloads" ItemName="TargetingPackForPruneDataCollated" />
+    </CollatePackageDownloads>
+
+    <!-- Can't run custom tasks in the build before NuGet restore, so create a separate project and restore that to download the packages -->
```

Comment:
When is this run?

Reply:
This is run during the build of the SDK repo.  We need to run restore in order to build the tasks which are used in building the repo, so we can't run custom tasks before NuGet restore.  So this moves the PackageDownload of the targeting packs to a separate project which we restore separately.

Reply:
Ah cool. 
The targeting pack solution, but at SDK build time.